package.json

```diff
@@ -207,6 +207,23 @@
           "type": "boolean",
           "default": true
         },
+        "codecov.fileDecorations.show": {
+          "description": "Whether to decorate each file in the file tree with a meter representing its coverage",
+          "type": "boolean",
+          "default": false
+        },
+        "codecov.fileDecorations.low": {
+          "type": "number",
+          "default": 70
+        },
+        "codecov.fileDecorations.high": {
+          "type": "number",
+          "default": 99
+        },
+        "codecov.fileDecorations.optimum": {
+          "type": "number",
+          "default": 100
+        },
```

Comment:
Do you imagine any user would want `optimum` to be lower than `low`? If not, we can simply call these `red`/`yellow`/`green` thresholds

Reply:
I think `optimum` should always be 100, that's inherent to code coverage (higher is better, not lower is better).

I think the current settings are still more semantic though because `low`/`high` has meaning to blind users while `red`/`yellow`/`green` does not.

But honestly I think this configuration is not as important. The ideal case would be that we can respect the value in codecov.yml, but I feel like we're starting to over-engineer this feature instead of being iterative.